underscore.js

```diff
@@ -84,18 +84,23 @@
     };
   };
 
+  var builtinIteratee;
```

Comment:
It does. But eslint is being a pain. (See earlier discussion)

On Tue, Mar 29, 2016, 12:20 Graeme Yeates <notifications@github.com> wrote:

> In underscore.js
> <https://github.com/jashkenas/underscore/pull/2480#discussion_r57752919>:
>
> > @@ -84,18 +84,23 @@
> >      };
> >    };
> >
> > +  var builtinIteratee;
>
> So this doesn't work correct?
>
> var builtinIteratee = _.iteratee = function(...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/underscore/pull/2480/files/5efa51ebda047dc4aef8f3680ec31e42159e41d7#r57752919>
>
-- 

Adam K (mobile)